docs/basic_usage.md

```diff
@@ -111,34 +73,36 @@ The following file is the entry point of your React app.
 
 import React from 'react';
 import ReactDOM from 'react-dom';
-import {Provider} from 'react-redux';
-import {renderRoutes} from 'react-router-config';
-import {ReduxBrowserIntlProvider} from '@foes/react-i18n-routing';
+import {I18nRoutingProvider} from '@foes/react-i18n-routing';
 import createHistory from 'history/createBrowserHistory';
 
-// There are previously implemented files
-import i18n from './i18n/index.js';
+import routes from './routing/config';
```

Comment:
@gorkalaucirica We are importing two `routes`. Remove the wrong one.

Reply:
@gorkalaucirica ping!

Reply:
@gorkalaucirica are you seeing this! XD

Reply:
Sorry, I just updated it